src/core/constants.js

```diff
@@ -236,6 +236,38 @@ export const SURVEY_ACTION_DISMISSED = 'Dismissed survey notice';
 export const SURVEY_ACTION_SHOWN = 'Shown survey notice';
 export const SURVEY_ACTION_VISITED = 'Visited survey';
 
+// Mapping of GA Categories to HCT compatible methods.
+// All the HCT methods must be under 20 chars
+// and should match /^[a-z0-9]{1}[a-z0-9_]+[a-z0-9]{1}$/i
+// See https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/events.html#limits
+// for more details.
+export const HCT_DISCO_CATEGORY = 'disco.interaction';
+const HCT_ADDON_CLICK = 'addon_click';
+const HCT_ADDON_ENABLED = 'enabled';
+const HCT_ADDON_INSTALLED = 'installed';
+const HCT_ADDON_INSTALL_CANCELLED = 'cancelled';
+const HCT_ADDON_DOWNLOAD_FAILED = 'download_failed';
+const HCT_ADDON_INSTALL_STARTED = 'install_started';
+const HCT_ADDON_NAVIGATION_CLICK = 'navigation_click';
+const HCT_ADDON_UNINSTALLED = 'uninstalled';
+
+export const HCT_METHOD_MAPPING = {
+  [CLICK_CATEGORY]: HCT_ADDON_CLICK,
+  [DISCO_NAVIGATION_CATEGORY]: HCT_ADDON_NAVIGATION_CLICK,
+  [INSTALL_EXTENSION_CATEGORY]: HCT_ADDON_INSTALLED,
+  [INSTALL_THEME_CATEGORY]: HCT_ADDON_INSTALLED,
+  [UNINSTALL_EXTENSION_CATEGORY]: HCT_ADDON_UNINSTALLED,
+  [UNINSTALL_THEME_CATEGORY]: HCT_ADDON_UNINSTALLED,
+  [ENABLE_EXTENSION_CATEGORY]: HCT_ADDON_ENABLED,
+  [ENABLE_THEME_CATEGORY]: HCT_ADDON_ENABLED,
+  [INSTALL_CANCELLED_EXTENSION_CATEGORY]: HCT_ADDON_INSTALL_CANCELLED,
+  [INSTALL_CANCELLED_THEME_CATEGORY]: HCT_ADDON_INSTALL_CANCELLED,
+  [INSTALL_STARTED_EXTENSION_CATEGORY]: HCT_ADDON_INSTALL_STARTED,
+  [INSTALL_STARTED_THEME_CATEGORY]: HCT_ADDON_INSTALL_STARTED,
+  [INSTALL_DOWNLOAD_FAILED_EXTENSION_CATEGORY]: HCT_ADDON_DOWNLOAD_FAILED,
+  [INSTALL_DOWNLOAD_FAILED_THEME_CATEGORY]: HCT_ADDON_DOWNLOAD_FAILED,
```

Comment:
Would it make sense to add tests for this mapping?

Reply:
In what sense?

Reply:
Making sure that both `INSTALL_DOWNLOAD_FAILED_THEME_CATEGORY` and `INSTALL_DOWNLOAD_FAILED_EXTENSION_CATEGORY` map to `HCT_ADDON_DOWNLOAD_FAILED`, etc.?